pubsub/cloud-client/README.rst

```diff
@@ -152,26 +154,26 @@ To run this sample:
     at https://cloud.google.com/pubsub/docs.
 
     positional arguments:
-      project               Your Google Cloud project ID
-      {list_in_topic,list_in_project,create,create-push,delete,update,receive,receive-custom-attributes,receive-flow-control,listen_for_errors}
+      project_id            Your Google Cloud project ID
+      {list_in_topic,list_in_project,create,create-push,delete,update,receive,receive-custom-attributes,receive-flow-control,receive-synchronously,receive-synchronously-with-lease,listen_for_errors}
         list_in_topic       Lists all subscriptions for a given topic.
         list_in_project     Lists all subscriptions in the current project.
         create              Create a new pull subscription on the given topic.
-        create-push         Create a new push subscription on the given topic. For
-                            example, endpoint is "https://my-test-
-                            project.appspot.com/push".
+        create-push         Create a new push subscription on the given topic.
         delete              Deletes an existing Pub/Sub topic.
         update              Updates an existing Pub/Sub subscription's push
                             endpoint URL. Note that certain properties of a
                             subscription, such as its topic, are not modifiable.
-                            For example, endpoint is "https://my-test-
-                            project.appspot.com/push".
         receive             Receives messages from a pull subscription.
         receive-custom-attributes
                             Receives messages from a pull subscription.
         receive-flow-control
                             Receives messages from a pull subscription with flow
                             control.
+        receive-synchronously
+                            Pulling messages synchronously.
+        receive-synchronously-with-lease
+                            Pulling messages synchronously with lease management
```

Comment:
What file do these two samples live in?

Reply:
They live in `pubsub/cloud-client/subscriber.py` ([link](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/pubsub/cloud-client/subscriber.py)). 

We haven't updated the README.md for some while.